### ISSUE TITLE
[doc] args 'elementwise_mean' can actually mean global mean in some nn.*loss modules and nn.functional.*loss

### ISSUE THREAD

**Issue by weiyangfb** [pytorch/pytorch#10034]
Args 'elementwise_mean' can actually mean global mean in some nn.*loss modules and nn.functional.*loss, for instance, [torch.nn.MultiLabelSoftMarginLoss](https://github.com/pytorch/pytorch/blob/bfe2aa093e17c84db863672ad446de064e47dc7e/torch/nn/modules/loss.py#L889-L894)
